extensions/mssql/src/features.ts

```diff
@@ -27,6 +29,52 @@ export class TelemetryFeature implements StaticFeature {
 	}
 }
 
+export class AccountFeature implements StaticFeature {
+
+	constructor(private _client: SqlOpsDataClient) { }
+
+	fillClientCapabilities(capabilities: ClientCapabilities): void {
+		// this isn't explicitly necessary
+	}
+
+	initialize(): void {
+		this._client.onRequest(contracts.SecurityTokenRequest.type, async e => {
```

Comment:
Just so it's clear, can you have the return type of this function here? `Promise<contracts.RequestSecurityTokenResponse | undefined>`

Reply:
Type def added.